.eleventy.js

```diff
@@ -3,7 +3,7 @@ const viteConfig = require('./vite.config.js');
 const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
 const util = require('util');
-const { default: vuePlugin } = require('@vitejs/plugin-vue');
```

Comment:
This isn't needed because we are importing it into `vite.config.js` instead, right?